CONTRIBUTING.md

```diff
@@ -20,19 +22,33 @@ Have you identified a reproducible problem in the AL compiler or developer tools
-- For new features in AL/Visual Studio Code language and tools, as well as Business Central in general, use [Business Central Ideas](https://aka.ms/bcideas). This will allow other members of the community to vote on your idea and will allow us to prioritize based accordingly.
-- For suggestions for new static analysis rules or improvements to existing static analysis rules, use [Business Central Ideas](https://aka.ms/bcideas).
-- For issues related to the extensibility of applications authored by Microsoft, such as the `System Application` or the `Base Application`, use the [ALAppExtensions](https://github.com/Microsoft/ALAppExtensions) repository.
-- For issues concerning versions of Business Central that are in mainstream support you should use the [support channel](#support).
+#### **Security vulnerabilities**
+Please follow the instructions in this [document](SECURITY.MD) to ensure that the issue is handled with the highest priority.
+
+#### **Bugs**
+For bugs in the latest version of the AL Language extension for Visual Studio Code or any of the static analysis tools that accompany the AL Language extension, use the current [GitHub repository](https://github.com/microsoft/al). You should always use the latest version of the AL Language extension when targeting any version of Business Central. 
+
+If you are targeting Dynamics NAV 2018 or older, you must use the AL extension provided in the DVD and use the [support channel](#support) to report issues.
+
+For issues concerning versions of Business Central that are in mainstream support you should use the [support channel](#support).
+
+#### **Extensibility of first-party AL applications**
+For issues related to the extensibility of applications authored by Microsoft, such as the `System Application` or the `Base Application`, use the [ALAppExtensions](https://github.com/Microsoft/ALAppExtensions) repository.
+
+#### **Feature requests and suggestions**
+For new features in AL/Visual Studio Code language and tools, as well as Business Central in general, use [Business Central Ideas](https://aka.ms/bcideas). 
+
+For suggestions for new static analysis rules or improvements to existing static analysis rules, use [Business Central Ideas](https://aka.ms/bcideas).
+
+Posting your suggestion on [Business Central Ideas](https://aka.ms/bcideas) will allow other members of the community to vote on your idea and will allow us to prioritize based accordingly.
 
 ### Look for an existing issue
 
-Before you create a new issue, please do a search in [open issues](https://github.com/microsoft/AL/issues) to see if the issue or feature request has already been filed.
+Before you create a new issue, please do a search in [open issues](https://github.com/microsoft/AL/issues) to see if the issue or feature request has already been filed, and add to that instead.
```

Comment:
Should this be more explicit? If this issue has already been filed, contribute to the existing issue.

Reply:
Actually, we are developing this point below. So maybe we should just remove what you added